lib/generated/intl/messages_all.dart

```diff
@@ -0,0 +1,62 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
+// This is a library that looks up messages for specific locales by
+// delegating to the appropriate library.
+
+// Ignore issues from commonly used lints in this file.
```

Comment:
Something is not right with this file. There should be many strings in here. 

You can choose to NOT commit this file, or commit the file with all strings in it.